interface-definitions/include/firewall/name-ipv6-name.xml.i

```diff
@@ -0,0 +1,666 @@
+<!-- include start from firewall/name-ipv6-name.xml.i -->
+<leafNode name="default-action">
```

Comment:
Please move the default-action node to a dedicated include as it is re-used fpr ipv4.

Reply:
That file "name-ipv6-name.xml.i" used for both ipv4 and ipv6. It included 2 times
```
set nfirewall name
set nfirewall ipv6-name 
```